app/src/main/java/app/nexd/android/ui/helper/transcript/TranscriptViewModel.kt

```diff
@@ -98,34 +105,18 @@ class TranscriptViewModel(private val api: Api) : ViewModel() {
             .firstOrError()
             .observeOn(mainThread())
             .subscribe({
-                call.value = it.firstOrNull()
+                val call = it.firstOrNull()
+                if (call == null) {
+                    callError.value = R.string.transcript_error_empty_calls
+                }
+                this.call.value = call
             }, {
                 error.value = R.string.error_message_unknown // TODO: use proper error message
             })
 
         loadArticles()
     }
 
-    private fun resetData() {
-        call.value = null
-
-        firstName.value = null
-        lastName.value = null
-        street.value = null
-        number.value = null
-        zipCode.value = null
-        city.value = null
-        firstNameError.value = null
-        lastNameError.value = null
-        streetError.value = null
-        numberError.value = null
-        zipCodeError.value = null
-        cityError.value = null
-        error.value = null
-
-        articles.value = emptyList()
-    }
-
```

Comment:
why was this helper method removed?

Reply:
because the data should only be reset if a new transcription is requested. So only if transcriptCall() is called, the data is cleared. This method is called in the TranscriptLoadingFragment where errors could be printed (e.g. if no calls available to transcript).